docker/fusionauth/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   db:
-    image: postgres:12.9
+    image: postgres:16.0-bookworm
```

Comment:
I don't think we should use this version. We still are fixing some known issues with PostgreSQL 15, and I have no idea if 16 works. I assume it does, but I think it would be wise to stay on 14 or 15 for now.

Reply:
We updated all the example applications to this and though there was a blip of an issue, we haven't seen anything else. https://github.com/FusionAuth/fusionauth-issues/issues/2597 is the issue, @mark-robustelli looked into it more deeply than I did.

We also state we support postgresql 10+ in our system requirements doc.

Reply:
I guess in theory I don't care.. but it seems riskier than it needs to be. Up to you.

Reply:
@robotdan Are there some things specifically I should be testing for in 16?  The quickstarts seem to be working OK.

Reply:
@robotdan no features in 16 needed, just upgrading to latest major release. 

Going to 14 still buys us almost [3 years before it is deprecated](https://www.postgresql.org/support/versioning/).

I do think the issues I've seen around postgresql seem to be maintenance mode related, where a super user is creating the FusionAuth tables ( https://github.com/FusionAuth/fusionauth-issues/issues/2264 and others listed on that bug), but we skip that with docker, since we preconfigure the db connection.

I have a mild preference for version 16 in docker compose just for consistency.